components/test-suite/src/loci/tests/testng/FormatReaderTest.java

```diff
@@ -1123,6 +1123,18 @@ public void testDeltaT() {
     for (int i=0; i<reader.getSeriesCount(); i++) {
       config.setSeries(i);
 
+      // Test image acquisition date
+      String expectedDate = config.getDate();
+      String date = null;
+      if (retrieve.getImageAcquisitionDate(i) != null) {
+        date = retrieve.getImageAcquisitionDate(i).getValue();
+      }
+      if (expectedDate != null && date != null && !expectedDate.equals(date)) {
```

Comment:
Should we also fail in the instance when expectedDate is non null but actual date is null?